packages/expo-cli/src/commands/build/BuildError.ts

```diff
@@ -1,9 +1,8 @@
-import ExtendableError from 'es6-error';
-
-export default class BuildError extends ExtendableError {
+export default class BuildError extends Error {
   name = 'BuildError';
 
   constructor(public message: string) {
     super();
   }
 }
+BuildError.prototype.name = BuildError.name;
```

Comment:
You could skip this because `name` is set using a property initializer above (see line 2). Maybe property initializer would be a good way to do this across the board – makes it super clear what the value will be.

In built-in `Error` the name property seems to be non-enumerable, so to match that we’d have the use `defineProperty` instead, though. But I’m not too concerned about that, a regular property should be fine.